CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Make sure that we only declare a Service of type LoadBalancer as deployed after its IP address is published. [#547](https://github.com/Shopify/kubernetes-deploy/pull/547)
 - Add more validations to `RunnerTask`. [#554](https://github.com/Shopify/kubernetes-deploy/pull/554)
 - Validate secrets with `--server-dry-run` on supported clusters. [#553](https://github.com/Shopify/kubernetes-deploy/pull/553)
+- Add resource type `PersistentVolumeClaim` to prune whitelist. [#573](https://github.com/Shopify/kubernetes-deploy/pull/573)
```

Comment:
This should explicitly be marked as a breaking change and include instructions on how to verify whether your cluster is affected and how to exempt resources from pruning. See https://github.com/Shopify/kubernetes-deploy/blob/master/CHANGELOG.md#0270 for a starting point.

Reply:
Also I think you need to rebase. The changes showing in "next" above were already released.

Reply:
Thanks @KnVerey , that's really a good remind. I'll add it as breaking change and include instructions for it

Reply:
I think [Validate secrets with `--server-dry-run`](https://github.com/Shopify/kubernetes-deploy/pull/573/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR10) and [PersistentVolumeClaim in whitelist](https://github.com/Shopify/kubernetes-deploy/pull/573/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12) should be in `0.29.0 enchantments`.